coq/coq_file.py

```diff
@@ -161,6 +164,15 @@ def __validate(self):
                     step.diagnostics.append(diagnostic)
                     break
 
+    def __init_coq_version(self):
+        output = subprocess.check_output("coqtop -v", shell=True)
+        coq_version = output.decode("utf-8").split("\n")[0].split()[-1]
+        outdated = version.parse(coq_version) < version.parse("8.18")
+
+        # We ignore the tags [VernacSynterp] and [VernacSynPure]
```

Comment:
Add a comment talking about the versions. We ignore the tags for 8.18 and we tested the "outdated" for 8.17